packages/tables/frontend/sirius-components-tables/src/table/row/ResizeRowHandler.types.ts

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2024 CEA LIST.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { GQLLine, TableProps } from '../TableContent.types';
+
+export interface ResizeRowHandlerProps extends TableProps {
+  row: GQLLine;
```

Comment:
I don't think it's a good idea to extend `TableProps`
In the future, there's no reason why these two types shouldn't diverge.

Reply:
Done